src/main/scala/org/fortysevendeg/sparksftp/ReadingSFTPConnectorApp.scala

```diff
@@ -28,47 +28,81 @@ object ReadingSFTPConnectorApp extends IOApp {
           config.spark.serializer.contains("KryoSerializer").toString
         )
         .set("spark.hadoop.fs.sftp.impl", "org.apache.hadoop.fs.sftp.SFTPFileSystem")
+        .set("spark.sql.legacy.allowCreatingManagedTableUsingNonemptyLocation","true")
```

Comment:
Why are we using legacy config settings?

Reply:
Sometimes, It does not happen always, if a process have been cancelled, there can be files tied to the database in the storage that have been locked , and when the next job kicks off, the table creation can fail, with that setting, it can override it, without the need to manually delete that files.

Reply:
Is there any other setting superseding this legacy one?

Reply:
I have not found it.